.github/workflows/cd-app.yml

```diff
@@ -0,0 +1,36 @@
+name: Continuous Deployment - App
+# Need to set a default value for when the workflow is triggered from a git push,
+# which bypasses the default configuration for inputs and cannot use env.ENVIRONMENT
+# since env context is not accessible in this context
+run-name: Deploy ${{ github.ref_name }} to ${{ inputs.environment || 'dev' }}
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "app/**"
+      - "bin/**"
+      - "infra/**"
```

Comment:
can you add a `!!` comment here and comment this out then update https://github.com/navapbc/template-infra/blob/main/template-only-docs/set-up-cd.md to reference this file instead of cd.yml